### PR TITLE
Add page reordering

### PR DESCRIPTION
**Pull request by ConnorCimowsky**

### CODE REVIEW

File: src-frontend/app/components/page-list.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    didInsertElement: function() {
+        var self = this;
+        this.$("ul.procedure").sortable({
+            items: 'li:not(:first-child)',
+            handle: 'span.drag-handle',
+            revert: true,
+            scroll: true,
+            placeholder: 'ui-state-highlight',
+            update: function(event, ui) {
+                var indices = {};
```
Comment (Trinovantes) [inline src-frontend/app/components/page-list.js]: Shouldn't this be an array?

File: src-frontend/app/components/page-list.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+export default Ember.Component.extend({
+    didInsertElement: function() {
+        var self = this;
```
Comment (Trinovantes) [inline src-frontend/app/components/page-list.js]: should just call this emberComponent

**Comment by Kapin** [comment]
Non-CSS/LESS makes sense. lgtm

**Comment by Trinovantes** [comment]
+1

**Comment by Ommy** [comment]
+1 (y)

**Pull request merged**